### PR TITLE
Codemod to NextJS 15

### PR DESCRIPTION
**Pull request by smthomas**
Upgrades admin to next15

### CODE REVIEW

File: packages/admin/src/domains/workflows/components/workflow-graph/workflow-popup-actions-bar.tsx [hunk 1]
```diff
@@ -121,8 +121,8 @@ export const WorkflowPopupActionsBar = ({ scale, setScale }: WorkflowPopupAction
           selectedValues={selectedScale}
           setSelectedValues={setSelectedScale}
           onSelectItem={val => {
+            console.log('onssssss');
```
Comment (smthomas) [inline packages/admin/src/domains/workflows/components/workflow-graph/workflow-popup-actions-bar.tsx]: Can we remove this log?

File: packages/admin/src/lib/framework-utils.ts [hunk 2]
```diff
@@ -2,20 +2,36 @@ import { Mastra } from '@mastra/core';
...
+  await fse.outputFile(outputFilePath, configFileString);
+}
+
+async function getFramework() {
   try {
-    const { config } = require(process.env.CONFIG_PATH!);
+    await loadConfig();
 
-    const framework = Mastra.init(config);
+    const { config } = require('../../temp-mastra-config.ts');
```
Comment (smthomas) [inline packages/admin/src/lib/framework-utils.ts:24]: Interesting. I'm guessing this has something to do with how compilation/bundling happens? If we hardcode the path then it works correctly, but we need to dynamically load the config in a different way (by reading the file and writing it to a static location)?

File: packages/admin/src/domains/workflows/components/workflow-graph/workflow-popup-actions-bar.tsx (hunk 1 above)
Comment (taofeeq-deru) [reply packages/admin/src/domains/workflows/components/workflow-graph/workflow-popup-actions-bar.tsx]: oh yeah, missed that

File: packages/admin/src/lib/framework-utils.ts (hunk 2 above)
Comment (taofeeq-deru) [reply packages/admin/src/lib/framework-utils.ts:24]: yeah
if i run require on the CONFIG_PATH directly, it fails with this error ` server relative imports are not implemented yet. Please try an import relative to the file you are importing from.`
so what i do is first read the file and write to a new temp file, and then run require on that new temp file path. Also the temp file is git ignored, so it's never pushed.

**Review by taofeeq-deru — APPROVED** [review]

**Pull request merged**